### PR TITLE
Fixes bad/deprecated use of checkurl

### LINKED ISSUE #27: Bad use of checkurl ?
Hello,

I don't have time to write a full issue description, but please have a look at : https://forum.yunohost.org/t/installation-of-2-community-apps-fails/3668/10

### PR DESCRIPTION
**Pull request by selamanse**
If ok this Closes #27

### CODE REVIEW

**Review by alexAubin — APPROVED** [review]
Not tested, but the part about checking the availability of the URL and booking it looks alright 👍

**Comment by selamanse** [comment]
thx @alexAubin. I'm currently running tests on the #30 issue and will later run the tests on this one.

File: scripts/install [hunk 1]
```diff
@@ -82,8 +85,8 @@ sed -i "s@#DESTDIR#@$final_path@g"  ../conf/nginx.conf
 sudo cp ../conf/nginx.conf "$nginx_conf"
 
 # download and extract rocketchat
-echo "Downloading rocket.chat-$ROCKETCHAT_VERSION.gtar from https://rocket.chat/releases/${ROCKETCHAT_VERSION}/download."
-sudo curl -s -L -o $final_path/rocket.chat-$ROCKETCHAT_VERSION.gtar "https://rocket.chat/releases/${ROCKETCHAT_VERSION}/download"
+echo "Downloading rocket.chat-$ROCKETCHAT_VERSION.gtar from https://download.rocket.chat/build/rocket.chat-${ROCKETCHAT_VERSION}.tgz."
```
Comment (dicer) [inline scripts/install:88]: That is already its own pull request, right? Same for upgrade

Comment (selamanse) [reply scripts/install:88]: correct, I wanted this change to be dependend on the other one

**Review by dicer — APPROVED** [review]

**Pull request merged**